pkgs/top-level/all-packages.nix

```diff
@@ -25961,7 +25961,9 @@ in
 
   tusk = callPackage ../applications/office/tusk { };
 
-  trufflehog = callPackage ../tools/security/trufflehog { };
+  trufflehog = callPackage ../tools/security/trufflehog { 
+    pythonPackages = python3Packages;
```

Comment:
The nix expression is easier to read, if you just use python3Packages within the package.